### ISSUE TITLE
GGMAX BUG hurt fall doesn't kill the player !

### ISSUE THREAD

**Issue by imothep85** [TheGameCreators/GameGuruRepo#5544]
It's a bug with the FALL option in the game who hurt or kill the player.
When i fall from a bridge NOTHING happen, the player is not killed by the fall of the bridge.
BUT if i keep walking he suddenly die....

i TRIED every option...the player is not killed, i put his health to 100, and hurt multiplier to 500, i tried 5, 10, 20 etc
but still the player is not killed.
I put the hurt-fall to 100 nothing change, so i put to 10 nothing changes, i want that when the player fall even if its from 10m height, he die !
but it's impossible to be killed by falling.

![fall don't kill01](https://github.com/TheGameCreators/GameGuruRepo/assets/8946215/ab24e3c3-6843-4468-934f-256bf283da3c)

**Comment by imothep85** [comment]
Here the video showing that bug !
When i wait some second and JUMP the player DIE.
When i immediately jump the player don't die !

https://github.com/TheGameCreators/GameGuruRepo/assets/8946215/f5b420b8-94ea-4b78-a3bf-ec5890fcb5f2

**Comment by Kraven3D** [comment]
It appears the player is invincible for a few seconds after spawning. Whether or not this is intentional, couldn't say, but an on/off option added to the start marker would be nice because I can see where it would and wouldn't have it's uses. An option to set the duration of invincibility would also be useful.

Video of the same thing happening when taking explosive damage (meaning this isn't only related to falling):
https://www.youtube.com/watch?v=urI8_qvY_BU

**Comment by LeeBamberTGC** [comment]
@Kraven3D Yes this is intended to protect the player for the first few seconds, and when you have a health hud it should turn a different colour when you are in this initial invincible mode.
